rir/src/runtime/Code.cpp

```diff
@@ -0,0 +1,56 @@
+#include "Code.h"
+#include "utils/Pool.h"
+#include "ir/CodeEditor.h"
+#include "ir/BC.h"
+
+namespace rir {
+Code::Code(SEXP ast, unsigned cs, unsigned sourceSize, unsigned csl,
+           unsigned offset, bool isDefaultArg) {
+    magic = CODE_MAGIC;
```

Comment:
`magic` is already initialized in the header file.

Reply:
right